package.json

```
@@ -34,6 +37,7 @@
     "@babel/preset-react": "7.16.7",
     "@storybook/addon-essentials": "6.4.12",
     "@storybook/react": "6.4.12",
+    "@svgr/cli": "^6.2.1",
```

```suggestion
    "@svgr/cli": "6.2.1",
```
I think this would fix the current failure with the CI. The carat needs to be removed.

---

I since changed my PR to use the SVGR Node library but your suggestion fixed the issue